lib/kubernetes-deploy/kubeclient_builder.rb

```diff
@@ -28,7 +29,7 @@ def build_v1beta1_kubeclient(context)
     end
 
     def _build_kubeclient(api_version:, context:, endpoint_path: nil)
-      config = Kubeclient::Config.read(ENV.fetch("KUBECONFIG"))
+      config = GoogleConfig.read(ENV.fetch("KUBECONFIG"))
```

Comment:
What about just `ConfigWithGoogle` ? I'm just not a huge fan of having `KubernetesDeploy::KubeclientBuilder::KubeconfigWithGoogleSupport`